as_pacmod.dbc

```diff
@@ -514,6 +514,32 @@ BO_ 1030 STEERING_MOTOR_RPT_3: 8 PACMOD
  SG_ TORQUE_OUTPUT : 7|32@0- (0.001,0) [-2048.0|2048.0] "N-m"  CUSTOMER_ECU
  SG_ TORQUE_INPUT : 39|32@0- (0.001,0) [-2048.0|2048.0] "N-m"  CUSTOMER_ECU
 
+BO_ 1051 NOTIFICATION_CMD: 8 CUSTOMER_ECU
+ SG_ BUZZER_MUTE : 0|1@0+ (1,0) [0|1] ""  PACMOD
+ SG_ UNDERDASH_LIGHTS_WHITE : 1|1@0+ (1,0) [0|1] ""  PACMOD
+
+BO_ 1049 LINEAR_ACCEL_RPT: 8 PACMOD
+ SG_ LATERAL_NEW_DATA_RX : 0|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LONGITUDNAL_NEW_DATA_RX : 1|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ VERTICAL_NEW_DATA_RX : 2|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LATERAL_VALID : 3|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LONGITUDNAL_VALID : 4|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ VERTICAL_VALID : 5|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU 
+ SG_ LATERAL_ACCEL : 8|16@0- (0.01,0) [-327.68|327.67] "m/s^2"  CUSTOMER_ECU
```

Comment:
The "engineering unit" values of these ranges are non-sensical.  Did we just want to convey that these are signed int16 values?

Reply:
These are what we agreed to when we were all sitting around the table.